src/constants/index.ts

```diff
@@ -1,4 +1,4 @@
-export const WPOKT_ADDRESS = '0x12e84B1b8718e06C7b57dE6fa3b0E30d30Db22d6';
+export const WPOKT_ADDRESS = '0x12e84B1b8718e06C7b57dE6fa3b0E30d30Db22d6'.toLowerCase();
 export const WPOKT_DECIMALS = 6;
-export const TOKEN_GEYSER_ADDRESS = '0x32d5802C24892B53cFB00bfbba9dcfb74031594C';
+export const TOKEN_GEYSER_ADDRESS = '0x32d5802C24892B53cFB00bfbba9dcfb74031594C'.toLowerCase();
```

Comment:
Adding these `.toLowerCase()` because we seem to actually be checksum-sensitive due to the subgraph. 😅